cherrytrack/helpers/plates.py

```diff
@@ -0,0 +1,70 @@
+from cherrytrack.helpers.mysql import create_mysql_connection_engine
+from flask import current_app as app
+from sqlalchemy import select, and_
+
+from cherrytrack.models import SourcePlateWell, DestinationPlateWell
+import logging
+
+
+logger = logging.getLogger(__name__)
+
+
+def get_samples_for_source_plate(source_barcode):
+    logger.info("Attempting to get samples from source_plate_wells with the source barcode")
+
+    # assign db_connection to avoid UnboundLocalError in 'finally' block, in case of exception
+    db_connection = None
+
+    try:
+        sql_engine = create_mysql_connection_engine(app.config["RO_CONN_STRING"], app.config["DATABASE_NAME"])
```

Comment:
We should try use the db connection from `__init__.py` instead of redefining it here.